cms/models.py

```diff
@@ -92,19 +93,34 @@ def _get_child_page_of_type(self, cls):
     @property
     def products(self):
         page_data = []
+        past_data = []
         for page in self.featured_products.all():
             if page.course_product_page:
                 product_page = page.course_product_page.specific
                 run = product_page.product.first_unexpired_run
-                page_data.append(
-                    {
-                        "title": product_page.title,
-                        "description": product_page.description,
-                        "feature_image": product_page.feature_image,
-                        "start_date": run.start_date if run is not None else None,
-                        "url_path": product_page.get_url(),
-                    }
-                )
+                run_data = {
+                    "title": product_page.title,
+                    "description": product_page.description,
+                    "feature_image": product_page.feature_image,
+                    "start_date": run.start_date if run is not None else None,
+                    "url_path": product_page.get_url(),
+                }
+                if run and run.start_date and run.start_date < now_in_utc():
+                    past_data.append(run_data)
+                else:
+                    page_data.append(run_data)
+
+        # sort future course run in ascending order
+        page_data = sorted(
```

Comment:
Nit: wondering if this variable name should be changed to reflect that it contains future concourses. For example, `future_data` similar to `past_data`